packages/web/src/pages/upload-page/fields/ReleaseDateModalForm.tsx

```diff
@@ -28,15 +28,14 @@ export const ReleaseDateModalForm = () => {
   const [{ value }, , { setValue }] =
     useField<EditFormValues[typeof RELEASE_DATE]>(RELEASE_DATE)
 
-  const initialValues = useMemo(
-    () => ({
-      [RELEASE_DATE]: value ?? null
-    }),
-    [value]
-  )
+  const initialValues = useMemo(() => {
+    const initialValues = {}
+    set(initialValues, RELEASE_DATE, value)
```

Comment:
just double checking the reason we need this is cause RELEASE_DATE could be a `a.b` type key?

Reply:
yes, I'm using it because useField supports lodash-style nested field names
https://formik.org/docs/api/field#name